pkg/ccl/changefeedccl/helpers_test.go

```diff
@@ -912,7 +912,7 @@ func expectErrCreatingFeed(
 	if err == nil {
 		t.Errorf("No error from %s", create)
 	} else {
-		require.Contains(t, err.Error(), errSubstring)
+		require.Regexp(t, errPattern, err.Error())
```

Comment:
You actually don't need to do a regex, it should be sufficient that the full error contains your error as a substring, i.e. `require.Contains` is fine